src/_base/harness/scripts/mutagen.sh

```diff
@@ -43,17 +43,17 @@ setup_sync_container()
     local CONTAINER_VOLUME_MAPPINGS
     local line
     for CONTAINER_NAME in "${CONTAINER_NAMES[@]}"; do
-        if [[ "$(docker ps -a -f "name=${CONTAINER_NAME}" --format '{{.Names}}')" == "${CONTAINER_NAME}" ]]; then
-            passthru docker rm -f "${CONTAINER_NAME}"
+        if [[ "$(docker ps --all --filter "name=${CONTAINER_NAME}\$" --format '{{.Names}}')" == "${CONTAINER_NAME}" ]]; then
+            passthru docker start "${CONTAINER_NAME}"
+        else
+            CONTAINER_VOLUME_MAPPINGS=()
+            while IFS= read -r line; do
+                CONTAINER_VOLUME_MAPPINGS+=("$line")
+            done < <(echo "${VOLUME_MAPPINGS}" | grep "^${CONTAINER_NAME}" | cut -d ':' -f2- )
+
+            # shellcheck disable=SC2046
+            passthru docker run -d --name "${CONTAINER_NAME}" $(printf -- '-v %q ' "${CONTAINER_VOLUME_MAPPINGS[@]}") alpine:latest /bin/sh -c "'trap : TERM INT; sleep 9999999999d & wait'"
```

Comment:
the "' '" is a bit odd, is it still a bug to do with workspace version?

Reply:
seeing as we're soon getting that released (driven by php 8 update rather than our other changes), we prob need something compatible

Reply:
Yes my `~/.my127/workspace/lib/sidekick.sh` is the old version at the moment.

Reply:
and I'm guessing compatible way is only via passthru "..."

Reply:
```suggestion
            passthru docker run -d --init --name "${CONTAINER_NAME}" $(printf -- '-v %q ' "${CONTAINER_VOLUME_MAPPINGS[@]}") alpine:latest tail -f /dev/null
```
tini from docker to the rescue?

Reply:
(just not available from docker-compose v3 file format)